C/Coin-OR/coin-or-common.jl

```diff
@@ -76,7 +76,7 @@ SHOT_version = offset_version(v"1.1.0", v"0.0.0")
 Julia_compat_version = "1.6"
 ASL_version = v"0.1.3"
 METIS_version = v"5.1.2"
-MUMPS_seq_version = v"500.500.101"
```

Comment:
Can you add a `MUMPS_seq_version_LBT` such we keep track of this version?

Reply:
Not sure what you mean - like just another version that is called `MUMPS_seq_version_LBT` that is 500.500.101, but is not used anywhere?

Reply:
Yes, it's what I mean.
It is not used anywhere but we should use it in the `build_tarballs.jl` of `Ipopt_jll`.